website/docs/docs/dbt-versions/release-notes/02-Nov-2023/explorer-updates-rn.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Enhancement: New features and UI changes to dbt Explorer"
+description: "November 2023: New features and UI changes to dbt Explorer"
+sidebar_label: "Enhancement: New features and UI changes to dbt Explorer"
+sidebar_position: 08
+tags: [Nov-2023]
+---
+
+dbt Labs is excited to announce the latest features and UI updates to dbt Explorer!
+
+For more details, refer to [Explore your dbt projects](/docs/collaborate/explore-projects).
+
+## The project's lineage graph 
+
+- The search bar in the full lineage graph is now to the right of the graph and is larger. 
+- The project/account navigation breadcrumbs has been updated.
+- A context menu is now available so you can explore a node's lineage and its details more easily from the full lineage graph.
+
+<LoomVideo id='1d28cc6443a94f00bbcb43d6ab3ad0d9' />
+
+## Search improvements 
+
+- When searching with keywords, a new side panel UI to filter search results by resource type. 
```

Comment:
- When searching with keywords, a new side panel UI helps filter search results by resource type, tag, column, and other key properties (in lieu of manually defining selectors).